### PR TITLE
Fe 1255 multi action button

### PR DESCRIPTION
**Pull request by grabkowski**
Updated Multi Action Button to be DLS compliant.

### CODE REVIEW

File: src/components/multi-action-button/multi-action-button.spec.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  });
+
+  describe('when rendered with "classic" theme', () => {
+    it('should match the snapshot', () => {
+      expect(renderWithTheme({
+        carbonTheme: classicTheme
+      }, TestRenderer.create)).toMatchSnapshot();
+    });
+
+    it('should match the snapshot', () => {
```
Comment (izakrzewska) [inline src/components/multi-action-button/multi-action-button.spec.js]: I think it's good to add that it's the test for 'transparency' prop, right now it looks like a duplicate of a previous test

File: src/components/multi-action-button/multi-action-button.stories.js [hunk 2]
```diff
@@ -2,60 +2,93 @@ import React from 'react';
...
-import Button from '../button';
+import { ThemeProvider } from 'styled-components';
+import MultiActionButton from './multi-action-button.component';
+import Button, { OriginalButton } from '../button';
 import OptionsHelper from '../../utils/helpers/options-helper';
-import { StoryHeader, StoryCode, StoryCodeBlock } from '../../../.storybook/style/storybook-info.styles';
+import { notes, info } from './documentation';
+import classic from '../../style/themes/classic';
+
+const getIconKnobs = () => {
```
Comment (izakrzewska) [inline src/components/multi-action-button/multi-action-button.stories.js]: icon knobs seems not to be visible in a story knobs

**Comment by harpalsingh** [comment]
Component looks fine when reviewing default and themes, but a couple of issues in classic/classic theme:

multi

**Comment by harpalsingh** [comment]
Default with Medium theme and the text is smaller:

Screenshot 2019-06-13 at 16 12 13

The child box should expand the buttons, but do we also pass a max-width to allow multi-line?

**Comment by harpalsingh** [comment]
Could we add 4 more pixels of padding to Classic version? I know the carbon has 10px as right padding, but it should be 14px to keep the balance of 18px each side @grabkowski

File: src/components/multi-action-button/multi-action-button-classic.style.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+import StyledSplitButton from '../split-button/split-button.style';
+import StyledButton from '../button/button.style';
+import StyledSplitButtonChildrenContainer from '../split-button/split-button-children.style';
+
+const getMultiActionButtonClassicStyles = ({ disabled, displayed, theme }) => (theme.name === THEMES.classic ? css`
+  ${StyledSplitButton} > ${StyledButton},
+  ${StyledSplitButton} > ${StyledButton}:focus {
+    margin: 0;
+    height: 31px;
+    padding: 7px 10px 7px 18px;
```
Comment (harpalsingh) [inline src/components/multi-action-button/multi-action-button-classic.style.js]: could we make this 7px 14px 7px 18px as per my feedback

Comment (grabkowski) [reply src/components/multi-action-button/multi-action-button-classic.style.js]: changed
